core/src/main/scala/spire/std/byte.scala

```diff
@@ -59,7 +59,7 @@ trait ByteOrder extends Order[Byte] {
 
 trait ByteSigned extends Signed[Byte] with ByteOrder {
   override def signum(a: Byte): Int = java.lang.Integer.signum(a)
-  override def abs(a: Byte): Byte = (if (a < 0) -a else a).toByte
+  override def abs(a: Byte): Byte = (if (a < 0) -a else a: Int).toByte
```

Comment:
in scala 3 byt is not converted to int automatically

Reply:
I think it's better to use `toInt` here, the scala compiler knows to do proper widening, it's not an actual method call.

Reply:
You're right. `toInt` works best avoiding any boxing

Reply:
Actually, can we just do like my comment here? It seems silly to widen only to narrow again.

https://github.com/typelevel/spire/pull/1067#discussion_r711660390

Reply:
Agree, this is better, I didn't take a look at the wider context.